common/utils/src/main/resources/error/error-classes.json

```diff
@@ -1975,6 +1981,12 @@
     },
     "sqlState" : "42K0E"
   },
+  "INVALID_NAME_PARAMETERIZED_QUERY_ALL_PARAMETERS_MUST_BE_NAMED" : {
```

Comment:
This is too long, please, make it shorter. For instance: `MISSED_PARAMETER_NAME`